styles.css

```diff
@@ -379,3 +387,183 @@ form input[type=submit] {
   text-align: center;
   color: #42526e;
 }
+
+/* Media Queries */
+
+@media only screen and (min-width: 768px) {
+  body {
+    background:
+      url(images/desktopBg.png) no-repeat,
+      url(images/deskFormBg1.png) no-repeat left 0% top 3422px,
+      url(images/deskFormBg2.png) no-repeat left 19% top 3584px,
+      url(images/deskFormBg3.png) no-repeat right 0% top 3278.3px;
+    background-size: cover, 200px, 102px, 214px;
+  }
+
+  header nav,
+  header a {
+    display: none;
+  }
+
+  header ul {
+    display: flex;
+    position: absolute;
+    top: 16px;
+    left: 50%;
+    transform: translate(-50%);
+    text-align: center;
+    align-items: center;
+    justify-content: space-between;
+    width: 255px;
+  }
+
+  .intro {
+    all: unset;
+    position: absolute;
+    top: 232px;
+    left: 50%;
+    transform: translate(-50%);
+    width: 63.89%;
+    background-color: transparent;
+  }
+
+  .intro h1 {
+    all: unset;
+    height: 114px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    text-align: center;
+    letter-spacing: 0.37px;
+    background-color: transparent;
+  }
+
+  .intro p {
+    all: unset;
+    height: 96px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    background-color: transparent;
+  }
+
+  #line-break::after {
+    content: "\a";
+    white-space: pre;
+  }
+
+  #line-break::before {
+    content: "\a";
+    white-space: pre;
+  }
+
+  .icons {
+    position: absolute;
+    top: 490px;
+    background-color: transparent;
+  }
+
+  .action {
+    position: absolute;
+    top: 604px;
+  }
+
+  #portfolio {
+    position: absolute;
+    top: 1139px;
+    left: 50%;
+    transform: translate(-50%);
+    grid-template-columns: 33% 33% 33%;
+    width: 1171px;
+    column-gap: 0;
+    background-color: transparent;
+  }
+
+  #portfolio h3 {
+    grid-column-start: 1;
+    grid-column-end: 4;
+  }
+
+  #about {
+    position: absolute;
+    top: 2411px;
+    left: 50%;
+    transform: translate(-50%);
+    width: 100%;
+  }
```

Comment:
- [ ] Nice work following the instructions of the last code reviewer and fixing the mobile version, but know the button is overlapping the paragraph in the desktop version, please 🙏🏽 fix it for a better user interface.
![image](https://user-images.githubusercontent.com/58642949/208160265-616683d6-1d0a-469c-a881-39b020ddded3.png)